src/file.ts

```diff
@@ -22,10 +28,16 @@ if (url.searchParams.has("from-file") && "launchQueue" in window && "files" in L
   dialog.showModal();
 }
 
+if (!("showOpenFilePicker" in window) || !("showSaveFilePicker" in window)) {
+  openButton.disabled = true;
+  createButton.disabled = true;
+  alert("File tools are not supporeted by current browser.");
```

Comment:
It might be better to be more specific so that users can figure out what browsers are supported.

Reply:
Ah good point. I have it in the error message but not where the user is